test/puppetlabs/puppetdb/testutils/jetty.clj

```diff
@@ -17,11 +17,12 @@
   "Creates a default config, populated with a temporary vardir and
    a fresh hypersql instance"
   []
-  {:repl {}
+  {:nrepl {}
```

Comment:
I changed this because with the `adjust-tk-config` now being used in the tests we see a the deprecation warning in our tests.